src/main/java/ch/njol/skript/expressions/ExprName.java

```diff
@@ -71,6 +70,17 @@
 	
 	private static final boolean inventoryTitles = Skript.methodExists(Inventory.class, "getTitle");
 	
+	@Nullable
+	private static final Method TITLE_METHOD;

```

Comment:
Use MethodHandle here instead.

Reply:
Okay, I made the change, but Im not 100% confident in my changes as Ive never used MethodHandle before. If you don't mind looking over and letting me know of any needed changes I would appreciate it.
I've tested on both 1.13.2 and 1.14.2 to make sure this works in both instances and it's working just great.